Makefile

```
@@ -20,13 +25,35 @@ GOLANGCILINT_VERSION = 1.49.0
 RUNNING_IN_CI = true
 -include build/makelib/golang.mk
 
+# ====================================================================================
 # Setup Kubernetes tools
+
+UP_VERSION = v0.13.0
+UP_CHANNEL = stable
 -include build/makelib/k8s_tools.mk
 
+# ====================================================================================
 # Setup Images
-DOCKER_REGISTRY ?= crossplane
-IMAGES = $(PROJECT_NAME) $(PROJECT_NAME)-controller
--include build/makelib/image.mk
+
+IMAGES = provider-ansible
+-include build/makelib/imagelight.mk
+
+# ====================================================================================
+# Setup XPKG
+
+XPKG_REG_ORGS ?= xpkg.upbound.io/crossplane index.docker.io/crossplane
```

This is a recent update :)
```suggestion
XPKG_REG_ORGS ?= xpkg.upbound.io/crossplane-contrib index.docker.io/crossplanecontrib
```

---

Yes I saw your post on slack :)